builder/vsphere/supervisor/step_create_source.go

```diff
@@ -39,14 +44,18 @@ type CreateSourceConfig struct {
 	// Name of the storage class that configures storage-related attributes.
 	StorageClass string `mapstructure:"storage_class" required:"true"`
 
-	// Name of the source VM. Defaults to `packer-vsphere-supervisor-built-source`.
+	// Name of the source VM. Defaults to `packer-vsphere-supervisor-<random-suffix>`.
 	SourceName string `mapstructure:"source_name"`
 	// Name of the network type to attach to the source VM's network interface. Defaults to empty.
 	NetworkType string `mapstructure:"network_type"`
 	// Name of the network to attach to the source VM's network interface. Defaults to empty.
 	NetworkName string `mapstructure:"network_name"`
 	// Preserve the created objects even after importing them to the vSphere endpoint. Defaults to `false`.
 	KeepInputArtifact bool `mapstructure:"keep_input_artifact"`
+	// Name of the bootstrap provider to use for configuring the source VM. Defaults to `CloudInit`.
```

Comment:
It would helpful to list all supported bootstrap here.

Reply:
Updated in the latest commit.